cloudify_handler/cloudify.py

```diff
@@ -29,10 +35,21 @@ def _bind(self):
            to set auto_delete=True)
         """
         credentials = pika.PlainCredentials(self.user, self.password)
```

Comment:
what happens when you pass `None` to `user` and `password`, because this is what we are doing here if `bootstrap_context.cloudify_agent` is not configured